get-img.py

```diff
@@ -1,11 +1,11 @@
 import json, requests
 
 with open("token.txt") as f:
-    token = f.readline()
+    token = f.readline().strip()
 
 def updateperson(person):
 	if not person["github"]:
-		person["github"] = "segerphilip"
+		person["github"] = "olinjs"
```

Comment:
There's probably a yet better fallback than this but this is certainly a better one than pip (nothing against pip).